simapp/README.md

```diff
@@ -4,50 +4,123 @@ sidebar_position: 1
-    two names can be anything, but you will need to use the same "chain id" in the following steps.
-4. `$ ./simd keys add [key_name]`. This will create a new key, with a name of your choosing.
-    Save the output of this command somewhere; you'll need the address generated here later.
-5. `$ ./simd genesis add-genesis-account [key_name] [amount]`, where `key_name` is the same key name as
-    before; and `amount` is something like `10000000000000000000000000stake`.
-6. `$ ./simd genesis gentx [key_name] [amount] --chain-id [chain-id]`. This will create the genesis
-    transaction for your new chain. Here `amount` should be at least `1000000000stake`. If you
-    provide too much or too little, you will encounter an error when starting your node.
-7. Now, one person needs to create the genesis file `genesis.json` using the genesis transactions
-   from every participant, by gathering all the genesis transactions under `config/gentx` and then
-   calling `$ ./simd genesis collect-gentxs`. This will create a new `genesis.json` file that includes data
-   from all the validators (we sometimes call it the "super genesis file" to distinguish it from
-   single-validator genesis files).
-8. Once you've received the super genesis file, overwrite your original `genesis.json` file with
-    the new super `genesis.json`.
-9. Modify your `config/config.toml` (in the simapp working directory) to include the other participants as
-    persistent peers:
-
-    ```text
-    # Comma separated list of nodes to keep persistent connections to
-    persistent_peers = "[validator_address]@[ip_address]:[port],[validator_address]@[ip_address]:[port]"
-    ```
-
-    You can find `validator_address` by running `$ ./simd comet show-node-id`. The output will
-    be the hex-encoded `validator_address`. The default `port` is 26656.
-10. Now you can start your nodes: `$ ./simd start`.
+Except stated otherwise, all participants in the testnet must follow through with each step.
+
+### 1. Download and Setup
+
+Download the Cosmos SDK and unzip it. You can do this manually (via the GitHub UI) or with the git clone command.
+
+```python
+git clone github.com/cosmos/cosmos-sdk.git
+```
+
+Next, run this command to build the `simd` binary in the `build` directory.
+
+```python
+make build
+```
+
+If you’ve run `simd` in the past, you may need to reset your database before starting up a new testnet. You can do that with this command:
+
+```python
+# you need to provide the moniker and chain ID
+$ ./simd init [moniker] --chain-id [chain-id]
+```
+
+The command should initialize a new working directory at the `~simapp` location. 
+
+The `moniker` and `chain-id` can be anything but you need to use the same `chain-id` subsequently.
+
+You can cut the hassles of specifying the directory by making the `simd` binary executable from all directories. 
+
+Open your bash profile file, add this command, and save the file.
```

Comment:
We can expect people to have their GOPATH properly configured, let's remove this.

Reply:
If someone is new to Go, trying to work with this, we can hardly expect that ser
Installing Go on macos hardly sets that up most times too. 
But sure, we can do that

Reply:
Someone new to Go should you at our step b step tutorials: https://tutorials.cosmos.network/hands-on-exercise/0-native/1-preparation.html#
IMHO here isn't the place.

Reply:
that's chill; we could link to it

Reply:
removed! 🫡